ArduCopter/mode_land.cpp

```diff
@@ -77,9 +82,12 @@ void ModeLand::gps_run()
         if (land_pause && millis()-land_start_time >= LAND_WITH_DELAY_MS) {
             land_pause = false;
         }
-
-        land_run_horizontal_control();
-        land_run_vertical_control(land_pause);
+#if PRECISION_LANDING == ENABLED
+        // the state machine takes care of the entire landing procedure
+        run_precland();
```

Comment:
I guess that we've lost the pause before landing because it isn't sent into run_precland()?